### ISSUE TITLE
command '[object Object]' not found

### ISSUE THREAD

**Issue by davidwolfpaw** [kneely/vscode-note-macros#7]
When running a note macro via a keyboard shortcut, I get the error `command '[object Object]' not found` displayed. The following is the error log after the run:

```
[2022-05-22 05:11:05.974] [exthost] [error] Error: command '[object Object]' not found
    at S._tryExecuteCommand (vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1694:3532)
    at S.executeCommand (vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1694:3414)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) note-macros.weeklyNote {"value":"NeelyInnovations.note-macros","_lower":"neelyinnovations.note-macros"}
```

If I use the command palette to run the macro, the error is not logged or displayed. Only when using a keyboard shortcut. I have tried multiple shortcut options, and it happens with any shortcut that I use.

**Comment by fanlushuai** [comment]
same problem !

**Comment by fanlushuai** [comment]
> When running a note macro via a keyboard shortcut, I get the error `command '[object Object]' not found` displayed. The following is the error log after the run:
> 
> ```
> [2022-05-22 05:11:05.974] [exthost] [error] Error: command '[object Object]' not found
>     at S._tryExecuteCommand (vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1694:3532)
>     at S.executeCommand (vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1694:3414)
>     at processTicksAndRejections (node:internal/process/task_queues:96:5) note-macros.weeklyNote {"value":"NeelyInnovations.note-macros","_lower":"neelyinnovations.note-macros"}
> ```
> 
> If I use the command palette to run the macro, the error is not logged or displayed. Only when using a keyboard shortcut. I have tried multiple shortcut options, and it happens with any shortcut that I use.

There is a bug on the code that executes the wrong command.
The reason seems to be that the shortcut key trigger does not catch the exception, while the ordinary command trigger catches the exception.

I don't know when it will be merged and packaged for temporary use.
https://github.com/fanlushuai/note-macros/releases/download/v0.0.3.fix/note-macros-0.0.3.vsix
